### PR TITLE
add new filter params - active_only and created_since

### PR DESCRIPTION
**Pull request by gowabash**
Documentation update for [CH62506](https://app.clubhouse.io/lessonly/story/62506/api-add-the-ability-to-filter-out-archived-users-from-the-list-users-endpoint)

Adds 2 new filters.  

Check for spelling mistakes and bad grammar.  I suck at those.

### CODE REVIEW

File: source/includes/_users.md.erb [hunk 1]
```diff
@@ -149,7 +149,7 @@ This endpoint retrieves all users.
 Parameter | Required | Type | Description
 --------- | ------- | ------- | -----------
 <%= pagination_query_params.chomp %>
-filter | no | String | Specified user filter for users list. Supported filters are email, ext_uid, sso_id, and custom_user_field. The SSO ID is the provider ID that we store from a successful SSO request. The format for custom_user_field is `filter[custom_user_field][FIELD_ID]=FIELD_VALUE`.
+filter | no | String | Specified user filter for users list. Supported filters are email, ext_uid, sso_id, custom_user_field, created_since and active_only. The SSO ID is the provider ID that we store from a successful SSO request.  `created_since` takes a date format of "YYYY-MM-DD".  `active_only` returns only active users of the value "true" is passed.  Anything else will result in all users being returned.  The format for custom_user_field is `filter[custom_user_field][FIELD_ID]=FIELD_VALUE`.
```
Comment (stevegrossi) [inline source/includes/_users.md.erb:152]:
```suggestion
filter | no | String | Specified user filter for users list. Supported filters are email, ext_uid, sso_id, custom_user_field, created_since and active_only. The SSO ID is the provider ID that we store from a successful SSO request.  `created_since` takes a date format of "YYYY-MM-DD".  `active_only` returns only active users if the value "true" is passed.  Anything else will result in all users being returned.  The format for custom_user_field is `filter[custom_user_field][FIELD_ID]=FIELD_VALUE`.
```

Comment (tmaconnelly) [inline source/includes/_users.md.erb:152]: I think we want : `active_only returns only active users of the value "true" is passed`
to read:
`active_only returns only active users if the value "true" is passed`

**Review by stevegrossi — CHANGES_REQUESTED** [review]
Looks like one typo? Also, I'm not able to run this locally to test it out, are you?

Screen Shot 2021-02-01 at 10 01 22 AM

**Comment by gowabash** [comment]
I was able to run it locally.

**Review by stevegrossi — APPROVED** [review]
Looks good 👍 I mentioned in Slack that based on the README for this repo it doesn't look like updates are published automatically. Will you handle publishing once ready?

**Comment by gowabash** [comment]
That was my plan once the other branch made it to production

**Pull request merged**